Popmax (#895)
broadinstitute/gnomad-browser: browser/help/topics/popmax.md

---
id: popmax
title: 'Population maximum'
---

> This annotation contains allele frequency information (AC, AN, AF, homozygote count) for the outbred population with the highest frequency. 

Reviewer: ```suggestion
This annotation contains allele frequency information (AC, AN, AF, homozygote count) for the non-bottlenecked population with the highest frequency. 
```

> For v2, this excludes Ashkenazi Jewish (asj), European Finnish (fin), and "Other" (oth) populations.

Reviewer: ```suggestion
For gnomAD v2, this excludes Ashkenazi Jewish (asj), European Finnish (fin), and "Other" (oth) populations.
```

> For v3, this excludes Amish (ami), Ashkenazi Jewish (asj), European Finnish (fin), Middle Eastern (mid), and "Other" (oth) populations.

Reviewer: ```suggestion
For gnomAD v3, this excludes Amish (ami), Ashkenazi Jewish (asj), European Finnish (fin), Middle Eastern (mid), and "Other" (oth) populations.
```
